### ISSUE TITLE
The column 'id' was specified multiple times for 'X'.

### ISSUE THREAD

**Issue by walkingonbothsides** [michiya/django-pyodbc-azure#34]
Hi,

after Upgrading to Django 1.8 I get an error in the Django adminpanel if I browse to to a sub-page on a model-admin (parameter ?p=X). This happens on every model that has more than 1 page.

The error is "The column 'id' was specified multiple times for 'X'." (where 'id' changes on different models).

The query it is trying to run is for example:

```
u'SELECT * FROM (SELECT [fips_address].[id], [fips_address].[address], [fips_address].[address_hex], [fips_address].[subnet_id], [fips_address].[hostname], [fips_address].[state], [fips_address].[usage], [fips_address].[description], [fips_address].[owner_id], [fips_address].[last_change], [fips_subnet].[id], [fips_subnet].[netaddr], [fips_subnet].[netaddr_hex], [fips_subnet].[netmask], [fips_subnet].[gateway], [fips_subnet].[vlan], [fips_subnet].[dhcp], [fips_subnet].[dhcp_start], [fips_subnet].[dhcp_end], [fips_subnet].[usage], [fips_subnet].[owner_id], ROW_NUMBER() OVER (ORDER BY [fips_address].[id] DESC) AS [rn] FROM [fips_address] INNER JOIN [fips_subnet] ON ( [fips_address].[subnet_id] = [fips_subnet].[id] )) AS X WHERE X.rn BETWEEN 101 AND 200 ORDER BY X.rn'
```

Full Error is:

```
e ProgrammingError('42000', "[42000] [FreeTDS][SQL Server]The column 'id' was specified multiple times for 'X'. (8156) (SQLExecDirectW)")
self <sql_server.pyodbc.base.CursorWrapper object at 0x7fef9c8c6b10>
```

SQL-Server Studio returns the same error on the query.

I'm running two maschines: 

```
CentOS7 x86_64
Python 2.7.5
Django==1.8.2 (same with 1.8.1)
django-pyodbc-azure==1.8.1.0
pyodbc==3.0.10

Debian 7 x86_64
Python 2.7.3
Django==1.8.2
django-pyodbc-azure==1.8.1.0
pyodbc==3.0.7

SQL-Server:
Microsoft SQL Server 2008 R2 (SP2) - 10.50.4033.0 (X64) 
    Jul  9 2014 16:04:25 
    Copyright (c) Microsoft Corporation
    Enterprise Edition (64-bit) on Windows NT 6.1 <X64> (Build 7601: Service Pack 1)
```

**Comment by daybarr** [comment]
I think this is a duplicate of issue #32 which has been fixed in commit 6c4e9b1 but not yet released to PyPI. That fix worked for me when I hit this same problem.

**Comment by walkingonbothsides** [comment]
Ah, thank you. Seems that I missed that issue :/
Just added this small change manual and now it works.
So I'm lokking forward for the final release.
